Doc/library/code.rst

```diff
@@ -22,6 +22,12 @@ build applications which provide an interactive interpreter prompt.
    it defaults to a newly created dictionary with key ``'__name__'`` set to
    ``'__console__'`` and key ``'__doc__'`` set to ``None``.
 
+   Note that functions and classes objects created under an
+   :class:`!InteractiveInterpreter` instance will belong to the namespace
+   defined by *locals*.
+   They are only pickleable if *locals* defined the namespace of an existing
+   module.
```

Comment:
```suggestion
   specified by *locals*.
   They are only pickleable if *locals* is the namespace of an existing
   module.
```
If locals is an existing namespace, I would not say it 'defines' the namespace.  I am not sure how this means to pass 'the namespace of an existing module'?  Just pass the import name, as in `import ast; ii = IteractiveInterpreter(ast)`?  If so, I would 'if locals is an existing module'.